nativeJS/brad_Go/momentum/app.js

```diff
@@ -1,58 +1,10 @@
-// HTML 요소를 JS로 가져오기
 const h1 = document.querySelector(".hello h1");
```

Comment:
```suggestion
const testHeader1 = document.querySelector(".hello h1");
```

변수명은 가능한, 구분되어질 수 있게 작성하는 습관을 들이는게 중요합니다!
arr, idx, temp 들 같은 의미를 알 수 없는 이름의 변수는 loop안에서 임시로 사용하시는거 아니면 가능한 사용을 피하시고, 더 명확한 의미의 변수명을 사용하는걸 추천드려요!